cogs/ReactForRole.py

```diff
@@ -689,9 +689,10 @@ async def overwrite_channel_add_reaction_perms(self, guild: discord.Guild, chann
         overwrite: discord.PermissionOverwrite = discord.PermissionOverwrite()
         overwrite.update(add_reactions=False)
         await channel.set_permissions(role, overwrite=overwrite)
-        bot_members: List[discord.Member] = [member for member in guild.members if member.bot]
-        for bot in bot_members:
-            await channel.set_permissions(bot, overwrite=None)
+        bot_members = [member for member in guild.members if member.bot and "Koala" in str(member)]
```

Comment:
checking for containing Koala isn't the best way and can be exploiited, maybe try something like member.bot and member.id == self.bot.id or something like that